content/templates.xqm

```diff
@@ -136,21 +138,26 @@ declare function templates:process($nodes as node()*, $model as map(*)) {
                 return
                     if ($dataAttr) then
                         templates:call($dataAttr, $node, $model)
-                    else
+                    else if (($model($templates:CONFIGURATION)($templates:CONFIG_USE_CLASS_SYNTAX), $templates:SEARCH_IN_CLASS)[1]) then
                         let $instructions := templates:get-instructions($node/@class)
                         return
                             if ($instructions) then
                                 for $instruction in $instructions
                                 return
                                     templates:call($instruction, $node, $model)
                             else
-                                element { node-name($node) } {
-                                    $node/@*, for $child in $node/node() return templates:process($child, $model)
-                                }
+                                templates:process-childs($node, $model)
+                    else templates:process-childs($node, $model)
             default return
                 $node
 };
 
+declare %private function templates:process-childs($node as node(), $model as map(*)) {
```

Comment:
Please rename new function to `templates:process-children`.

Reply:
Done.